src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -177,7 +178,8 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
             FunctionContext? context = null;
             InvocationResponse response = new()
             {
-                InvocationId = request.InvocationId
+                InvocationId = request.InvocationId,
+                Result       = new StatusResult()
```

Comment:
very nit: Not sure if there are extra spaces after Result or it is just rendered here like this.

Reply:
extra spaces are intentional to line up the `=` - easier to read :D